l3_msgs/Foothold foot
l3_footstep_planning_msgs/UpdateMode update_mode
---
l3_msgs/Foothold foot
l3_footstep_planning_msgs/ErrorStatus status
